src/components/Card.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { TaskStepCard, TaskStepCardProps } from './Card';
+import { Default as ExerciseQuestion } from './ExerciseQuestion.stories';
+
+const props: TaskStepCardProps = {
+  step: {
+    type: 'exercise' as const,
+    task: {
+      is_deleted: false,
+      type: 'homework' as const
+    },
+    uid: '1234@1',
+    id: 1,
+    available_points: '1.0',
+  },
+  questionNumber: 1,
+  numberOfQuestions: 1
+};
+
+export const Default = () => <TaskStepCard {...props}><ExerciseQuestion /></TaskStepCard>;
+export const CustomComponents = () =>
+  <TaskStepCard
+    {...props}
+    leftHeaderChildren={<button>◀</button>}
+    rightHeaderChildren={<>Two-step ℹ️ <button>▶</button></>}
+    headerTitleChildren="🔒"
+  >
```

Comment:
Tutor can use these to send the navigation and lock buttons, which uses the FontAwesome icon library. We may want to use something lighter on the assignment service side, so I'm making it a node prop for flexibility.